Nighttime-Operations/Weather-Constraints/index.rst

```diff
@@ -0,0 +1,179 @@
+If the direction changes frequently, or has completely changed direction from 300 degrees, use caution as this likely indicates a frontal system and unstable atmospheric conditions.
+
+Wind speeds between 0 and 12 m/s (gusts):
+    - This is safe operation range for AuxTel. 
+        No restrictions in observing are needed. 
+        
+Wind speeds between 12 and 15 m/s (gusts):
+    - If the observer has control to choose targets, observe out of the wind.
+        Report that the scheduler configuration needs to be changed, or the observing program should be changed. 
+        Avoid targets that are pointing direction into or 180 degrees opposite of the dominant direction of the wind.
+
+.. note::
+    90 degree rule:
+        Due to the curved dome, pointing at targets 180 degrees offset from the dominant wind direction creates a wind eddy under the lip of the dome, which can create wind shake on the upper truss of AuxTel. 
+        To compensate for this, it is recommended to look for targets that have a 90-degree offset from the dominant wind direction. 
+
+Wind speeds greater than 15 m/s (gusts):
+    - Observers should consider closing once wind gusts have become greater than 15 m/s. 
+        Sustained winds (gusts averaged over 5 minutes) of more than 15 m/s should initiate telescope closure. 
+        Before deciding to open again, ensure that sustained winds have dropped below the closure limits for more than 15 minutes. 
+
+.. _weather-constraints-humidity-and-dew-point:
+
+Humidity and dew point
+^^^^^^^^^^^^^^^^^^^^^^
+
+Higher relative humidity is not an absolute determination of closure, but observers must pay close attention when humidity is over 70%, or if it begins to suddenly trend upward.
+
+Humidity above 65%:
+    - Turn off the extraction fan and close vent gates.
+
+Humidity above 70%:
+    - Observers must treat this range with extra caution. 
+        Take time to go outside and inspect vehicles, buildings, and look for falling drops from the roof. 
+        Be mindful of local humidity changes between AuxTel and the main telescope building.
+        Because AuxTel is at a slightly higher elevation, there is a small buffer against condensation.
+        If the ambient temperature is cooler, especially during winter time, the condensation risk on metal surfaces is higher. 
+
+The quantity that is more important than relative humidity is the **dew point temperature** and the **dew point difference**. 
+
+The dew point temperature is the ambient air temperature at which relative humidity will reach 100% - the air is completely saturated with water vapor.
+
+The dew point difference is the difference between the dew point and the coldest structure in the telescope dome. 
+
+Example:
+    Dew point temperature = 8 degrees C.
+
+    AuxTel M1 temperature = 6 degrees C.
+    
+    Glycol lines = 4 degrees C.
+    
+    Dew point difference = 4 degrees C.
```

Comment:
To make this abundantly clear, it'd be better if no numbers were the same.

I'm a little confused here. The surfaces are all below the dew point temperature, which means they're all condensing. 
Would more realistic numbers be:
    Dew point temperature = 8 degrees C.

    AuxTel M1 temperature = 12 degrees C.

    Refrigeration lines = 10 degrees C.

    Dew point difference = 2 degrees C.

Then it would be good to add a sentence further down saying something like "In this case, the refrigeration lines will be condensing and the telescope should be closed"

Reply:
👍